cypress/e2e/tests/pages/users-and-auth/users.spec.ts

```diff
@@ -122,9 +107,8 @@ describe('Users', { tags: ['@usersAndAuths', '@adminUser'] }, () => {
     const mgmtUserEditPo = new MgmtUserEditPo();
 
     mgmtUserEditPo.globalRoleBindings().globalOptions().then((list) => {
-      expect(list.length).to.eq(4);
+      expect(list.length).to.eq(3);
       expect(list[0]).to.eq('Administrator');
-      expect(list[1]).to.eq('Restricted Administrator');
```

Comment:
After this change, all the other indexes must be updated accordingly, or?

Cypress dashboard: `fe43b3ff-a329-4c38-8d8e-cb81336c319d`

<img width="1507" alt="Screenshot 2024-11-27 at 15 53 03" src="https://github.com/user-attachments/assets/a25be1af-5fb9-4fe9-939a-e7836ebe4011">

Reply:
updated the PR